analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -141,7 +139,7 @@ object PythonVersion : CommandLineTool {
             // Parse a line like " -2.7-32        C:\Python27\python.exe".
             versionAndPath?.split(' ', limit = 3)?.last()?.trimStart()
         } else {
-            getPathFromEnvironment("python$version")?.path
+            Os.getPathFromEnvironment("python$version")?.path
```

Comment:
I couldn't find a way to comment on the previous commit "core-utils: Move all storage tests under functional tests": The test classes should also be renamed to end with "FunTest".

Reply:
Good catch, thanks, fixed.